style.css

```diff
@@ -0,0 +1,162 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500&display=swap");
+ 
+.container {
+background-color: rgba(255, 81, 0, 0.822);
+display: flex;
+width: 510px;
+height: 500px;
+margin-left: 350px;
+margin-top: 30px;
+border-radius: 15px;
+color: white;
+font-family: "Montserrat";
+ }
+
+.portfolio {
+    background-color: coral;
+    border-radius: 15px;
+    margin-left: 15px;
+    margin-top: 15px;
+    margin-bottom: 15px;
+    margin-right: 0;
+    width: 230px;
+    text-align: center;
+    
+}
+
+.profile_status {
+    border-radius: 15px;
+    margin-left: 30px;
+    margin-top: 15px;
+    margin-bottom: 15px;
+    margin-right: 0;
+    width: 220px;
+    text-align: center;
+    
+}
+
+.post_box {
```

Comment:
three blocks have common styles (`.post_box, .like_box, .follower_box`)

create one common class and add these styles to one class